README.md

```diff
@@ -1 +1,15 @@
-# Mobile-version-skeleton
\ No newline at end of file
+# Mobile-version-skeleton
+> This project is about  creating html and styling files in order to built a responsive Mobile version skeleton .
+## Built With
+- HTML
+- CSS
+- Git & Github
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+1. Open Terminal
+2. Clone the Project 
+3. Open browser using `live server`
+## Authors
+**LOIC**
+- GitHub: [@Hirwacardinalloic]
+- Twitter: [@loicard]
```

Comment:
- [ ] You should include all sections provided in the ReadMe [template](https://github.com/microverseinc/readme-template/blob/master/README.md) by Microverse. Therefore, kindly update your ReadMe accordingly for a professional look. Fo example, Sub-sections of the `Getting Started` section, `Acknowledgement`, `Contributions`, `License` sections are missing.